src/app/organization/offices/edit-office/edit-office.component.ts

```diff
@@ -0,0 +1,81 @@
+/** Angular Imports */
+import { Component, OnInit } from '@angular/core';
+import { FormGroup, FormBuilder, Validators } from '@angular/forms';
+import { ActivatedRoute, Router } from '@angular/router';
+import { DatePipe } from '@angular/common';
+
+/** Custom Services */
+import { OrganizationService } from 'app/organization/organization.service';
+
+/**
+ * Edit Office component.
+ */
+@Component({
+  selector: 'mifosx-edit-office',
+  templateUrl: './edit-office.component.html',
+  styleUrls: ['./edit-office.component.scss']
+})
+export class EditOfficeComponent implements OnInit {
+
+  /** Selected Data. */
+  officeData: any;
+  /** Office form. */
+  officeForm: FormGroup;
+  /** Date. */
+  openedOn: any;
+
+    /**
+     * Retrieves the charge data from `resolve`.
+     * @param {ProductsService} organizationService Organization Service.
+     * @param {FormBuilder} formBuilder Form Builder.
+     * @param {ActivatedRoute} route Activated Route.
+     * @param {Router} router Router for navigation.
+     * @param {MatDialog} dialog Dialog reference.
+     * @param {DatePipe} datepipe Convert Date.
+     */
+    constructor(private organizationService: OrganizationService,
+      private formBuilder: FormBuilder,
+      private route: ActivatedRoute,
+      private router: Router,
+      private datepipe: DatePipe
```

Comment:
indent till they reach bottom of organization service
```suggestion
         private formBuilder: FormBuilder,
         private route: ActivatedRoute,
         private router: Router,
         private datepipe: DatePipe
```